react/features/base/jwt/functions.ts

```diff
@@ -165,15 +197,28 @@ export function validateJwt(jwt: string) {
                         && featureValue !== 'true'
                         && featureValue !== 'false'
                     ) {
-                        errors.push(`- Invalid value for feature: ${feature}`);
+                        errors = {
+                            ...errors,
+                            [`- Invalid value for feature: ${feature}`]:
```

Comment:
This could be expressed as a `invalid-value-for-feature-THEFEATURE` constant, following my suggestion above.

Reply:
Are you sure? There are 15 of those atm..

Reply:
Why not? It's a dynamic field, we wouldn't have a dedicated constant for each bur taher a prefix, so if it starts with `invalid-value-for-feature-` we know what comes next.